core/src/main/scala/kafka/coordinator/group/GroupCoordinatorAdapter.scala

```diff
@@ -557,25 +557,25 @@ private[group] class GroupCoordinatorAdapter(
   }
 
   override def onElection(
-    partitionIndex: Int,
-    partitionLeaderEpoch: Int
+    consumerOffsetsPartitionIndex: Int,
+    consumerOffsetsPartitionLeaderEpoch: Int
   ): Unit = {
-    coordinator.onElection(partitionIndex, partitionLeaderEpoch)
+    coordinator.onElection(consumerOffsetsPartitionIndex, consumerOffsetsPartitionLeaderEpoch)
   }
 
   override def onResignation(
-    partitionIndex: Int,
-    partitionLeaderEpoch: OptionalInt
+    consumerOffsetsPartitionIndex: Int,
+    consumerOffsetsPartitionLeaderEpoch: OptionalInt
   ): Unit = {
-    coordinator.onResignation(partitionIndex, partitionLeaderEpoch)
+    coordinator.onResignation(consumerOffsetsPartitionIndex, consumerOffsetsPartitionLeaderEpoch)
   }
 
-  override def offsetsTopicConfigs(): Properties = {
+  override def consumerOffsetsTopicConfigs(): Properties = {
     coordinator.offsetsTopicConfigs
   }
 
-  override def startup(groupMetadataTopicPartitionCount: IntSupplier): Unit = {
-    coordinator.startup(() => groupMetadataTopicPartitionCount.getAsInt)
+  override def startup(consumerOffsetsPartitionCount: IntSupplier): Unit = {
```

Comment:
sorry to go back and forth on this one, but I wonder if we call it "GROUP_METADATA_TOPIC_NAME" in topics, if we should actually just call it groupMetadataTopic here (and above). It's still more specific than "partition" in the above cases.

Reply:
I don't think that we can use `groupMetadataTopic` as prefix everywhere because in onElection/onResignation we really want to expression the partition index. However, we could use `groupMetadata` as prefix in those cases.

I updated the PR, let me know what you think.